spyder_terminal/terminalplugin.py

```diff
@@ -58,6 +59,14 @@ class TerminalPlugin(SpyderPluginWidget):
 
     URL_ISSUES = ' https://github.com/spyder-ide/spyder-terminal/issues'
     CONF_SECTION = 'terminal'
+    CONFIGWIDGET_CLASS = TerminalConfigPage
+    CONF_DEFAULTS = [
```

Comment:
Let's move the contents of this variable to an external file called `config.py` (in the same directory of this file).

That file will contain two variables:

* `CONF_DEFAULTS`, which will be the same as what you have for `CONF_DEFAULTS` here.
* `CONF_VERSION`, which you need to set to `1.0.0`. That value is important in case we need to change some of these options in the future.

After doing that, you need to import and use those constants here like this:

```python
from spyder_terminal.config import CONF_DEFAULTS, CONF_VERSION

...

class TerminalPlugin(SpyderPluginWidget):
    ...

    CONF_DEFAULTS = CONF_DEFAULTS
    CONF_VERSION = CONF_VERSION
```

Reply:
@goanpeca I can't use CONF_VERSION here because it is not implemented in SpyderPluginWidget

Reply:
It's defined in `SpyderPlugin` and `SpyderPluginWidget` inherits from it, so it should be possible.